packages/visx-brush/src/Brush.tsx

```diff
@@ -58,6 +58,8 @@ export type BrushProps = {
   handleSize: number;
   /** Reference to the BaseBrush component. */
   innerRef?: React.MutableRefObject<BaseBrush | null>;
+  /** Prevent drag end on mouse leave from brush */
+  isUseWindowMoveEvents?: boolean;
```

Comment:
same comment throughout

```suggestion
  useWindowMoveEvents?: boolean;
```